src/modules/roc_audio/sample_spec.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2020 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_audio/sample_spec.h
+
+#ifndef ROC_AUDIO_SAMPLE_SPEC_H_
+#define ROC_AUDIO_SAMPLE_SPEC_H_
+
+#include "roc_packet/units.h"
+
+namespace roc {
+namespace audio {
+
+//! Bitmask of channels present in audio packet.
+typedef uint32_t channel_mask_t;
+
+//! Default sample rate, number of samples per second.
+const size_t DefaultSampleRate = 44100;
+
+//! Default channel mask.
+const channel_mask_t DefaultChannelMask = 0x3;
```

Comment:
All defaults should stay in one place, roc_pipeline/config.h, because we want to avoid spreading various hard-coded values across the code base and be sure that they're not duplicated.

Thus, default SampleSpec constructor may initialize fields with zero, or, better, we can remove default constructor at all, if it wouldn't create too much problems.

And config structs in roc_pipeline should initialize their SampleSpec fields with appropriate defaults. Only roc_pipeline should know about those defaults.